pom.xml

```diff
@@ -51,9 +51,23 @@ under the License.
         <developerConnection>scm:git:https://github.com/apache/flink-shaded.git</developerConnection>
     </scm>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <!-- Allows exec-maven-plugin to resolve snapshot plugin dependencies -->
+            <id>apache.snapshots.https</id>
+            <name>${distMgmtSnapshotsName}</name>
+            <url>${distMgmtSnapshotsUrl}</url>
+        </pluginRepository>
+    </pluginRepositories>
+
     <properties>
         <shading.prefix>org.apache.flink.shaded</shading.prefix>
         <netty.version>4.1.70.Final</netty.version>
+        <!-- The license check requires the artifactId to match the directory that the module resides in.
+             This is not the case for several modules in flink-shaded for legacy reasons.
```

Comment:
This is intentionally non-descriptive because frankly I don't see any logic in which modules include a version in the artifactId.
For example, flink-shaded-asm-9 includes the major ASM version, which is actually cool because you can depend on multiple flink-shaded-asm artifacts if needed. But for some reason we don't allow that for guava.